dist/index.html

```diff
@@ -70,7 +70,8 @@ <h3 class="mb-5">
         <div class="row">
             <div class="col-lg-10 mx-auto">
                 <h2>Wie zijn we</h2>
-                <p class="lead mb-5">Wij zijn een jonge, gemotiveerde groep die jouw webpagina maakt zoals u het wilt.</p>
+                <p class="lead mb-5">Wij zijn een jonge, gemotiveerde groep die jouw webpagina maakt zoals u het
+                    wilt.</p>
```

Comment:
zou ge die enter daar tussen kunnen wegdoen zoda da ook ni op de webpagina komt of komt da er ni op?